fatal-errors.json

```diff
@@ -643,6 +644,12 @@
       "categoryId": "other",
       "kbLinkId": "AllFilesHaveFailed",
       "canUserFix": false
+    },
+    "MissingGeometry": {
+      "description": " No geometry has been created for entity %s.",

```

Comment:
Please change the description as suggested in the email

Reply:
Updated description as per mail.

Reply:
I guess I should have been more specific about two single quotes - I meant '%s'.  Sorry about confusion, Rohit!